db/patients.ts

```diff
@@ -17,17 +17,31 @@ const getById = async (patientId: string, psychologistId: string): Promise<Patie
   }
 };
 
-const getAll = async (psychologistId: string): Promise<(Patient & {appointmentsCount: string})[]> => {
+const startCurrentUnivYear = (): string => {
+  const currentMonth = new Date().getMonth();
+  const currentYear = new Date().getFullYear();
+  if (currentMonth < 6) {
```

Comment:
6 ? I thought it was until end of august ?

Reply:
oh right ! :facepalm: